plugin/src/Logger.cpp

```diff
@@ -201,7 +201,24 @@ SCSAPI_VOID Logger::gameplay(const scs_telemetry_gameplay_event_t *event_info) {
         m_job.status = JobStatus::Cancelled;
     } else if (event_id == SCS_TELEMETRY_GAMEPLAY_EVENT_job_delivered) {
         m_job.status = JobStatus::Delivered;
-    } else if (event_id != SCS_TELEMETRY_GAMEPLAY_EVENT_player_fined) {
+    } else if (event_id == SCS_TELEMETRY_GAMEPLAY_EVENT_player_fined) {
+        if (m_job.status != JobStatus::OnJob) {
```

Comment:
Why only when OnJob?

Reply:
I assumed you would only want to get fines during a job.

Reply:
Truck position is also sent when not in job so why not to also send fines.